data/coalition/heliarch intro.txt

```diff
@@ -0,0 +1,1843 @@
+			`	You head back inside and continue exploring the cave system. You two find a few more of the hidden campsites, but all are as abandoned as the first one. You're alerted by the wind, snow, and ice whenever you're close to another exit.`
+			`	After a few hours, the storm finally subsides, and you both return to your ship. You leave to pick up Kirrie and Tugroob, who also come back empty-handed.`
+				accept
+			label bumpyblizzard
+			`	Iplora hesitates for a bit, then explains that you would probably get sick going through the storm now. She suggests you wait in the cave until it is safe for human passage.`
+			`	You insist, and she reluctantly gives in, telling you to stick close to her. She moves slowly so you have no problem keeping up, but time and again you keep falling over, tripping on the mounds the rocks have created. Another fall, and Iplora reaches her hand out to help you up once again.`
+			choice
+				`	(Take her hand.)`
+				`	(Try and dig through the snow to find something.)`
+					goto snowdig
+			label back
+			`	She helps you up, and you continue on your way down the hills. You get back to the forest, and she starts finding the trail back to the ship. Halfway through, the storm gets progressively weaker.`
+			`	When you two return to your ship, you leave to pick up Kirrie and Tugroob, who also come back empty handed.`
+				accept
+			label snowdig
+			`	You start clawing your way down the snow as Iplora asks you to stop and continue moving toward the ship. You continue, your hands getting colder as the snow steals the heat even through your gloves. You hit a hard surface and start clearing from there, only to see it was, in fact, just a boulder.`
+			`	"Satisfied? Return, we must. Come now," You turn to see her offering her hand once again.`
+			choice
+				`	(Take her hand.)`
+					goto back
+				`	(Head to another mound and try digging there.)`
+			label keepdig
+			`	You stand up and run to another spot where you fell over. Iplora follows you, and even with the snow muffling the sounds around you, you can hear her exasperated sigh. She tries to pull you away, but you hit a hard surface yet again, this time creating a faint metallic echo. She takes her hands off of you and lays down, now helping you dig out whatever is there. Once the crate is visible, she stands up. It's large and looks like a coffin, but much of it is still buried. She asks you to help her get some hooks on it, after which she struggles to try and pull it out.`
+			`	The slippery snow certainly doesn't help her hooves, but it does make the crate slippery as well. It's coaxed out of the ground, bit by bit. You help her as best as you can and eventually get it on top of the snow field. Iplora searches through some tools in her bag and takes out what looks like a crowbar. She starts opening it, a dim light creeping out, but an energy blast coming from the the top of the hills hits the ground near the hole. It misses you by just about a meter.`
```

Comment:
Oh so now the crates are metal, also looks like a human coffin? If not, do Coalition species even use coffins? Would you know what one looked like even if they did?
Additionally, hooves are pretty good at snow, metal crates are the opposite of slippery when they're this cold, and if this crate is frozen you don't have a hope in hell of opening it with a crowbar, unless its internally powered somehow, in which case why would it need a crowbar to open?

Reply:
It looks like a coffin to the player, I don't see Iplora saying anything about the shape

Reply:
From the player's perspective it appears to be a coffin, yeah.